proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -49,6 +49,7 @@ message OrganizationUser {
   string organization_id = 2;
   // The role that the user has in the organization above.
   OrganizationRole organization_role = 3;
+  OrganizationRoleSource organization_role_source = 4;
```

Comment:
Just curious, in which scenarios do we need to expose this in RPC responses?

Reply:
I am thinking the UI will want to see this to power scenarios where we can show why a user is part of an org (e.g., they are part of a group).

Reply:
I ask because we'd need to check in the backend everytime we're moving a user proto and populate that value.

Reply:
Yeah, I'll have to do this up.